cleanup_utils/load_balancer_cleanup.py

```diff
@@ -24,6 +24,10 @@
 import logging
 import os
 import pathlib
+from operator import itemgetter
+
+from consul import Consul
```

Comment:
I'm not sure this requirement is being installed (the cleanup requirements are on a separate file).

Reply:
The cleanup jobs is working, but I'll double-check it.